examples/python/Quantinuum_parametrized_gates.py

```diff
@@ -0,0 +1,49 @@
+# <div style="text-align: center;">
+# <img src="https://assets-global.website-files.com/62b9d45fb3f64842a96c9686/62d84db4aeb2f6552f3a2f78_Quantinuum%20Logo__horizontal%20blue.svg" width="200" height="200" /></div>
+
+# # How to Submit Parametrized Circuits
+
+# Parametrized circuits are common in variational algorithms. Pytket supports parameters within circuits via symbols. For more information, see [Symbolic Circuits](https://cqcl.github.io/pytket/manual/manual_circuit.html?highlight=paramet#symbolic-circuits) and [Symbolic Compilation](https://tket.quantinuum.com/examples/symbolics_example.html?highlight=fresh_symbol).
```

Comment:
The manual link should be -> https://tket.quantinuum.com/user-manual/manual_compiler.html#compiling-symbolic-circuits